apps/web/src/index.css

```
@@ -927,7 +928,46 @@ a {
 
 @utility macro-message-padding {
   @container (width < 936px) { padding-inline: calc(var(--spacing) * 4); }
-  @container (width < 640px) { padding-inline: calc(var(--spacing) * 2); }
+}
+
+@utility macro-thread-collapsed-row {
```

Stylelint (17.14.0)

[error] Unexpected unknown at-rule "`@utility`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)